components/Layout/DatePicker.tsx

```diff
@@ -1,4 +1,7 @@
+import React, {useEffect, useState} from 'react';
 import dayjs, { Dayjs } from 'dayjs'
+import 'dayjs/locale/de' // import German locale
+import 'dayjs/locale/en' // import English locale
```

Comment:
Remove comments.

Reply:
Fixed in https://github.com/TUM-Dev/TUMenu/pull/110/commits/972b7d8ca37be9e33adeda3e0367e6b305a155f4